### PR TITLE
adding code for lesson Coding with Java Part 3

### PR DESCRIPTION
**Pull request by guy9**

### CODE REVIEW

**Comment by guy9** [comment]
@tzach please review, I tested this and it works.

**Comment by tzach** [comment]
@guy9 why do this example needs a new Docker instance?
Can we reuse the same Java docker image?

**Comment by guy9** [comment]
@tzach yes, done.

**Comment by guy9** [comment]
@tzach please review/merge

**Comment by tzach** [comment]
@guy9 where can I find the instructions for running this example?
Its better to have a short README per directory to give some context

**Comment by guy9** [comment]
@tzach done, keep in mind that the Readme file is very general and I don't want it to be used instead of the U. lessons.

File: mms2/java/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Scylla in Docker for the Mutant Monitoring University Course - Java Lessons 1 2 and 3
+
+### Instructions for setting up a Scylla Cluster from this repo, prerequisites are a running 3 node cluster with the catalog/tracking keyspaces and tables. More info in the University Lessons. 
+
+```
+cd mms2/java/<dir name>
+docker build -t java-app .
```
Comment (tzach) [inline mms2/java/README.md]: I'm getting
```
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/tzach/project/guy-code-samples/mms2/java/java-app-ps/Dockerfile: no such file or directory
```

Comment (tzach) [reply mms2/java/README.md]: I suspect this line should run at `mms2/java/` not at `mms2/java/<dir name>`

File: mms2/java/java-datatypes/src/DataTypes_App.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                Cluster cluster = Cluster.builder().addContactPoints("scylla-node1", "scylla-node2", "scylla-node3").build();
+                Session session = cluster.connect("catalog");
+                createSchema(session);
+
+                String[] mutants = new String[] {"Jim Jefferies", "Bob Loblaw", "Bob Zemuda"};
+                for (int i=0; i != mutants.length; i++) {
+                        if(mutants[i] != null) {
+                                String[] first_name = mutants[i].split(" ");
+                                String last_name = first_name[1];
+                                FILE = new File("/opt/code/" + first_name[0] + "_" + last_name + ".png");
```
Comment (tzach) [inline mms2/java/java-datatypes/src/DataTypes_App.java]: `/opt/code/` should be `/opt/code/java-datatypes`

Comment (guy9) [reply mms2/java/java-datatypes/src/DataTypes_App.java]: Thanks, fixed and tested this

**Pull request merged**